lib/plugins/auto-load.js

```diff
@@ -3,40 +3,44 @@
 const _ = require('lodash');
 const debug = require('debug')('voxa');
 
+let defaultConfig = { loadByToken: false };
+
 function autoLoad(skill, config) {
-  skill.onRequestStarted((alexaEvent) => {
-    if (!config) {
-      throw Error('Empty config file');
-    }
+  if (!config) {
+    throw Error('Empty config file');
+  }
 
-    if (!config.adapter) {
-      throw Error('Empty adapter');
-    }
+  if (!config.adapter) {
+    throw Error('Empty adapter');
+  }
+
+  if (!_.isFunction(config.adapter.get)) {
+    throw Error('No get method to fetch data from');
+  }
 
-    if (config.loadByToken && !_.get(alexaEvent, 'session.user.accessToken')) {
+  defaultConfig = _.merge(defaultConfig, config);
+
+  skill.onRequestStarted((alexaEvent) => {
+    if (defaultConfig.loadByToken && !_.get(alexaEvent, 'session.user.accessToken')) {
       debug('Missing token on load data by token operation');
       return alexaEvent;
     }
 
     let key = _.get(alexaEvent, 'user.userId');
 
-    if (config.loadByToken) {
+    if (defaultConfig.loadByToken) {
```

Comment:
Well, the [alexa.user](http://voxa.readthedocs.io/en/latest/alexa-event.html)  is a documented interface and seems to me it's simple enough to just use it. 

I think that having an extra configuration key to decide what key to use doesn't really save you that much when you actually go and create a the adapter.  thoughts on this @rmberrios @heneryville ?